cli/setup.py

```diff
@@ -28,7 +28,7 @@ def readme():
     "tabulate>=0.8.2,<=0.8.3",
     "ipaddress>=1.0.22",
     "enum34>=1.1.6",
-    "PyYAML==5.2",
+    "PyYAML==5.2" if sys.version_info.major == 3 and sys.version_info.minor == 4 else "PyYAML>=5.2",
```

Comment:
Does PyYAML 5.3 not work only with Python 3.4 or even with previous version like 3.3?

Reply:
It doesn't work with any of 3.0-3.4. Here's what you get when you try to install 5.3 with python3.4:
```
❯ pip install pyyaml==5.3
DEPRECATION: Python 3.4 support has been deprecated. pip 19.1 will be the last one supporting it. Please upgrade your Python as Python 3.4 won't be maintained after March 2019 (cf PEP 429).
Collecting pyyaml==5.3
  Using cached https://files.pythonhosted.org/packages/3d/d9/ea9816aea31beeadccd03f1f8b625ecf8f645bd66744484d162d84803ce5/PyYAML-5.3.tar.gz
ERROR: PyYAML requires Python '>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*' but the running Python is 3.4.10
```

Reply:
https://github.com/yaml/pyyaml/blob/master/setup.py#L314

Reply:
Ok can we put `sys.version_info.minor <= 4` then?

Reply:
Sure thing.